README.md

```diff
@@ -38,11 +38,10 @@ that are not currently tested, can be found in [this spreadsheet](docs/PAS%20Req
 ## Test Scope and Limitations
 
 Neither the server nor client test suite included test the full scope of the PAS IG.
-Documentation within the tests include details on what is currently tested
-and what is out of scope and why. Start a session with either test kit or see the description in these files 
-([server](lib/davinci_pas_test_kit/generated/v2.0.1/server_suite.rb),
-[client](lib/davinci_pas_test_kit/client_suite.rb)) for details beyond what
-is included here.
+Documentation of what is currently tested and what is out of scope and why can be
+found in the suite descriptions when the tests are run, or within this repository
+for the [server](lib/davinci_pas_test_kit/docs/server_suite_description_v201.md#testing-limitations)
+and [client](lib/davinci_pas_test_kit/docs/client_suite_description_v201.md#testing-limitations).
 
 At a high-level, in-scope requirements include:
```

Comment:
Line 46 - 66: It would be good to have this info in distinct sections for better readability.

```suggestion
### In-Scope Requirements

At a high level, the following requirements are currently within the test scope:

- The ability of the system to participate in end-to-end prior authorization
  workflows, including those resulting in approval or denial, including cases
  where the request goes through a pending status to allow for additional time
  for the decision to be made.
- The ability of the system to produce and receive (currently server tests only)
  all PAS-defined profiles and their must support elements.

### Out-of-Scope Requirements

The following requirements, as outlined in the Implementation Guide, are currently out of scope and not yet tested:

- **Subscriptions**: [Details here](https://hl7.org/fhir/us/davinci-pas/STU2/specification.html#subscription)
- **Prior Authorization update workflows**: [Details here](https://hl7.org/fhir/us/davinci-pas/STU2/specification.html#updating-authorization-requests)
- **Requests for additional information handled through the CDex framework**: [Details here](https://hl7.org/fhir/us/davinci-pas/STU2/additionalinfo.html)
- **PDF, CDA, and JPG attachments**: [Details in 3rd paragraph here](https://hl7.org/fhir/us/davinci-pas/STU2/specification.html#prior-authorization-submission)
- **US Core profile support for supporting information**: [Details here](https://hl7.org/fhir/us/davinci-pas/STU2/specification.html#integration-with-other-implementation-guides)
- **Server-specific requirements**: 
  - Inquiry matching and subsetting logic: [Details in 2nd paragraph and 2 bullets here](https://hl7.org/fhir/us/davinci-pas/STU2/specification.html#prior-authorization-inquiries)
  - Inquiry requests from non-submitting systems: [Details in 3rd paragraph here](https://hl7.org/fhir/us/davinci-pas/STU2/specification.html#pended-authorization-responses)
  - Collection and dissemination of metrics: [Details here](https://hl7.org/fhir/us/davinci-pas/STU2/metrics.html)
- **Client-specific requirements**: 
  - The ability to handle responses containing all PAS-defined profiles and must support elements: [Details in 3rd bullet here](https://hl7.org/fhir/us/davinci-pas/STU2/background.html#must-support)
  - Most details requiring manual review of the client system, e.g., the requirement that clinicians can update details of the prior authorization request before submitting them: [Details in 1st paragraph here](https://hl7.org/fhir/us/davinci-pas/STU2/usecases.html#submit-prior-authorization)

```

Reply:
good idea - made similar tweaks - let me know if they look good.

Reply:
Looking good to me.